### PR TITLE
pass custom email parameters

### PR DESCRIPTION
**Pull request by brandonbryant12**

### CODE REVIEW

File: src/handcash_connect.ts [hunk 1]
```diff
@@ -106,8 +106,8 @@ export default class HandCashConnect {
 	 *
 	 * @returns {string} requestId - The request id.
```
Comment (rafa-js) [inline src/handcash_connect.ts:107]: Update the JSDoc for consistency 🙏

**Review by rafa-js — CHANGES_REQUESTED** [review]
👍

**Comment by brandonbryant12** [comment]
updated

**Review by rafa-js — APPROVED** [review]
👍👍

**Pull request merged**